pkg/embeddedyamls/yamls.go

```diff
@@ -40,7 +40,7 @@ spec:
   - name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: Broker is the Schema for the brokers API.
+        description: Broker is the Schema for the brokers API
```

Comment:
Do you mean to remove these periods? If so, please document why in the commit message.

Reply:
Also this is the generated version.

Reply:
This dots were added by 'make pkg/embeddedyamls/yamls.go'